lxd/api_internal.go

```diff
@@ -788,6 +789,10 @@ func internalImportFromBackup(d *Daemon, projectName string, instName string, fo
 		return err
 	}
 
+	// If the instance is a VM, generate a new `volatile.uuid.generation`
```

Comment:
The comment says "if the instance is a VM" but this isn't true, we do it for all instance types.

Reply:
I'd expect to see this set within CreateInternal, the same as `volatile.uuid` is.